SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -146,6 +150,21 @@ VirtualWorkerId ULayeredLBStrategy::WhoShouldHaveAuthority(const AActor& Actor)
 	return ReturnedWorkerId;
 }
 
+SpatialGDK::FActorLoadBalancingGroupId ULayeredLBStrategy::GetActorGroupId(const AActor& Actor) const
+{
+	const FName ActorLayerName = GetLayerNameForActor(Actor);
+
+	const int32 ActorGroupIndex = LayerData.FindChecked(ActorLayerName).LayerIndex + 1;
```

Comment:
Might be useful to have some utility function that transform layer index to actor group index and vice-versa. Helps keep to assumed equivalence clearer as well.